client/pages/tables.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import AppContext from '../lib/app-context';
+// import Redirect from '../components/redirect';
+import Navbar from '../components/navbar';
+import ListTables from '../components/listTables';
+
+export default class Tables extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tablesFetch: [],
+      isLoading: true,
+      isFreeModalSelected: false,
+      isAssignModalSelected: false,
```

Comment:
Using multiple booleans like this makes the code susceptible to lots of duplicated logic. Because only one modal should be open at any given time, i recommend changing from booleans to a string enum.

```js
openModal: // 'free' or 'assign' or null
```